discord/automod.py

```diff
@@ -177,35 +177,59 @@ class AutoModTriggerMetadata:
     ----------
     keyword_filter: List[:class:`str`]
         A list of substrings to filter. Only for triggers of type :attr:`AutoModTriggerType.keyword`.
+    regex_patterns: List[:class:`str`]
+        A list of regex patterns to filter. Only for triggeres of type :attr:`AutoModTriggerType.keyword`.
```

Comment:
```suggestion
        A list of regex patterns to filter. Only for triggers of type :attr:`AutoModTriggerType.keyword`.
```
Also this description should warn about the flavor of regex used is not entirely compatible with the builtin `re` module.

Reply:
I've fixed the typo and subsequent copy-paste, and added a warning about regex flavor. I initially didn't think a warning would be necessary, but I could see people attempting to use `re` to validate regex prior to updating on a guild.